test/type-resolution/__snapshots__/fixtures.test.ts.snap

```diff
@@ -6588,43 +6513,33 @@ TypedTemplate {
   },
   "resources": DependencyGraph {
     "_dependencies": Map {
-      "WebServerGroup" => Set {
-        "LaunchConfig",
-        "ALBTargetGroup",
-      },
-      "LaunchConfig" => Set {
-        "InstanceSecurityGroup",
-      },
       "ELBSecurityGroup" => Set {},
+      "ALBTargetGroup" => Set {},
+      "Dashboard" => Set {},
       "ApplicationLoadBalancer" => Set {
         "ELBSecurityGroup",
       },
       "ALBListener" => Set {
         "ALBTargetGroup",
         "ApplicationLoadBalancer",
       },
-      "ALBTargetGroup" => Set {},
       "InstanceSecurityGroup" => Set {
         "ApplicationLoadBalancer",
       },
       "RecordSet" => Set {
         "ApplicationLoadBalancer",
       },
-      "Dashboard" => Set {},
-    },
-    "keys": Set {
-      "ELBSecurityGroup",
-      "ALBTargetGroup",
-      "Dashboard",
-      "ApplicationLoadBalancer",
-      "ALBListener",
-      "InstanceSecurityGroup",
-      "RecordSet",
-      "LaunchConfig",
-      "WebServerGroup",
+      "LaunchConfig" => Set {
+        "InstanceSecurityGroup",
+      },
+      "WebServerGroup" => Set {
+        "LaunchConfig",
+        "ALBTargetGroup",
+      },
     },
     "nodes": Object {
       "ALBListener": Object {
+        "cfnType": "AWS::ElasticLoadBalancingV2::Listener",
```

Comment:
Added this to be able to refer to it an an error message. Not sure if it's really needed.